tiatoolbox/decorators/__init__.py

```diff
@@ -1,4 +1,4 @@
 """
-Decorators init file
+Package defines decorators for the toolbox
```

Comment:
Inconsistency with spacing at end of docstring- do we want to add blank space at the end of docstring like others?

Reply:
Single line docstrings will go on one line.